setup.py

```diff
@@ -95,14 +128,15 @@ def build_extensions(self):
         opts = self.c_opts.get(ct, [])
         link_opts = self.l_opts.get(ct, [])
         if ct == 'unix':
-            opts.append(cpp_flag(self.compiler))
-            if has_flag(self.compiler, '-fvisibility=hidden'):
-                opts.append('-fvisibility=hidden')
+            if sys.platform != 'darwin':
+                opts.append(cpp_flag(self.compiler))
```

Comment:
This condition was added when I used some `.c` file. I have removed that file by now. So this change can probably be removed.

Reply:
Fixed in https://github.com/letmaik/pyvirtualcam/pull/22.